### ISSUE TITLE
Embedding xsd schemas

### ISSUE THREAD

**Issue by andrew-hardin** [ngageoint/six-library#540]
There's a fair amount of brittle book keeping required to track the location of schemas and resolve them at run-time. An alternative approach involves embedding the schemas into the source (e.g. `xxd -i schema.xsd`), then at run-time passing those buffers to the xml validator.

Is that a feature that may be valuable to the broader community? If so, is it a feature that replaces the existing schema path machinery, or, does that path based machinery enable a valuable use case that I'm ignorant to?

**Comment by JDanielSmith** [comment]
Yes, the schema-path machinery is a bit complicated; I like the idea of embnedding a schema! But this probably needs to get run through "the process" (although I'm not sure exactly what that might be) as an enhancment.
